srtcore/core.cpp

```diff
@@ -2509,7 +1822,7 @@ bool CUDT::processSrtMsg(const CPacket *ctrlpkt)
             {
                 if (len_out == 1)
                 {
-                    if (m_bOPT_StrictEncryption)
+                    if (m_config.m_bOPT_StrictEncryption)
```

Comment:
Improve config field names. The `OPT_` infix is no longer necessary, as well as names can better represent the option.